src/index.tsx

```diff
@@ -1,7 +1,5 @@
-import "./../style/css/Styles.css";
-import "./../style/css/index.css";
-import "bootstrap/dist/css/bootstrap.min.css";
-
+import "../style/css/index.css";
+import { cell_utils, notebook_utils as nb_utils } from "./utils";
```

Comment:
multiple imports should be on multiple lines.

Reply:
I fixed what I can, but I have to update the Prettier settings because it keeps putting them on one line